modules/150-user-authn/template_tests/authenticator_test.go

```diff
@@ -75,9 +76,10 @@ var _ = Describe("Module :: user-authn :: helm template :: dex authenticator", f
     cookieSecret: cookieSecret
   allowAccessToKubernetes: true
   spec:
-    applicationDomain: authenticator.com
-    applicationIngressCertificateSecretName: test
-    applicationIngressClassName: test
+    applications:
```

Comment:
There is the reason to add more than one application for a single test case to check the render of the array of applications.